src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -176,10 +176,66 @@ public void initialize(URL location, ResourceBundle resources) {
         newMessageIndicator.visibleProperty().bind(newMessageIndicator.textProperty().isNotEqualTo("0"));
     }
 
+    @FXML
+    BorderPane quotaBlock;
+    @FXML
+    Label quota;
+    @FXML
+    Label provider;
+    @FXML
+    Label quotaDescription;
+    @Inject
+    BoxClient boxClient;
+
+    private void fillQuotaInformation() {
+        try {
+            quotaState = boxClient.getQuotaState();
+        } catch (IOException | QblInvalidCredentials e) {
+            alert(e);
+        }
+        if (quotaState == null) {
+            quotaBlock.setVisible(false);
+            quotaDescription.setVisible(false);
+            return;
+        }
+
+        int ratio = ratioByDiff(quotaState.getSize(), quotaState.getQuota());
+        String quotaDescriptionText = quotaDescription(quotaState.getSize(), quotaState.getQuota());
+        quota.setText(ratio + "%");
+        provider.setMinWidth(ratio);
```

Comment:
`provider` is not very descriptive, especially in this context. What does it represent? Please try to find a better name